gtsfm/loader/loader_base.py

```diff
@@ -43,15 +43,17 @@ def get_ui_metadata() -> UiMetadata:
             parent_plate="Loader and Retriever",
         )
 
-    def __init__(self, max_resolution: int = 1080) -> None:
+    def __init__(self, max_resolution: int = 1080, input_worker: str = None) -> None:
         """
         Args:
             max_resolution: integer representing maximum length of image's short side
                e.g. for 1080p (1920 x 1080), max_resolution would be 1080
+            input_worker: string representing ip address of the worker
```

Comment:
nit: can we add a period at the end of this line.

Reply:
Also, @stepanyanhayk you mentioned it needs the port address too? if so, could you add that to the docstring?